pkgs/development/libraries/wxwidgets/wxGTK31.nix

```diff
@@ -19,7 +19,7 @@
 , unicode ? true
 , withGtk2 ? (!stdenv.isDarwin)
 , withMesa ? lib.elem stdenv.hostPlatform.system lib.platforms.mesaPlatforms
-, withWebKit ? stdenv.isDarwin
+, withWebKit ? false
```

Comment:
Are there any working darwin uses for wxGTK? grass (which doesn't even have a working GUI because it's not using framework python) uses `wxmac` instead: https://github.com/NixOS/nixpkgs/blob/634141959076a8ab69ca2cca0f266852256d79ee/pkgs/applications/gis/grass/default.nix#L23-L26

Reply:
`kicad`, for example, doesn't even work on darwin:

```
$ nix-env -f $NIXPKGS -iA kicad
installing 'kicad-6.0.2'
error: Package ‘webkitgtk-2.34.6’ in /Users/wcohen/programming/nix/nixpkgs/pkgs/development/libraries/webkitgtk/default.nix:216 is marked as broken, refusing to evaluate.

       a) To temporarily allow broken packages, you can use an environment variable
          for a single invocation of the nix tools.

            $ export NIXPKGS_ALLOW_BROKEN=1

        Note: For `nix shell`, `nix build`, `nix develop` or any other Nix 2.4+
        (Flake) command, `--impure` must be passed in order to read this
        environment variable.

       b) For `nixos-rebuild` you can set
         { nixpkgs.config.allowBroken = true; }
       in configuration.nix to override this.

       c) For `nix-env`, `nix-build`, `nix-shell` or any other Nix command you can add
         { allowBroken = true; }
       to ~/.config/nixpkgs/config.nix.
(use '--show-trace' to show detailed location information)
```

They do note this in the derivation (https://github.com/NixOS/nixpkgs/blob/634141959076a8ab69ca2cca0f266852256d79ee/pkgs/applications/science/electronics/kicad/default.nix#L226-L230), so I assume that this means that getting that going would be to switch to `wxmac` instead of trying to fix `wxWidgets` using GTK on darwin

Reply:
Or put another way -- since webkitgtk is broken on darwin (https://github.com/NixOS/nixpkgs/pull/126101), and linux doesn't use it, there's not really a use case at the moment where `withWebKit` will evaluate to true, right?

Reply:
Ah I see it now @knl, you sidestepped webkitgtk on darwin via https://github.com/NixOS/nixpkgs/pull/163335/files#diff-fbc90381d9c2869d8bd91c09e8f3d90ac863518a20896cec053d464bf316c063R79-R80. Just updated to have this PR exclusively revert the assert.

Reply:
I confirmed that one can build wxwidgets 3.1.5 on both darwin and linux, and on darwin it works with webkit (in which case, it doesn't use webkitgtk) -- so why remove that?

Of course, many packages are broken on darwin, and we should work on fixing that situation. I just don't see how changing this default to false on darwin helps.